src/mmw/js/src/draw/views.js

```diff
@@ -568,6 +573,11 @@ var ResetDrawView = Marionette.ItemView.extend({
     },
 
     resetDrawingState: function() {
+        App.getLeafletMap().eachLayer((l) => {
+            if (l.id && l.id === 'rwd-origin-point') {
+                App.getLeafletMap().removeLayer(l);
+            }
+        });
```

Comment:
Makes sense. I amended that commit to store the marker in the model so that `get` and `unset` can be used in the reset button function.